### ISSUE TITLE
Nano: env variables are set different when keep using `source bigdl-nano-init`

### ISSUE THREAD

**Issue by plusbang** [intel-analytics/BigDL#6634]
When `conda activate`, nano env variables are set automatically. But if I keep using `source bigdl-nano-init` in the conda environment, the env variables `OMP_NUM_THREADS` and `NANO_TF_INTER_OP` are set different from before.
![image](https://user-images.githubusercontent.com/108676127/202097413-f99855e2-78f6-4215-852e-73652d37b42b.png)

I wonder why `OMP_NUM_THREADS` and `NANO_TF_INTER_OP` are set different in the two cases.

**Comment by MeouSker77** [comment]
```bash
	if [ -z "${ENABLE_TF_OPTS:-}" ]; then
		echo "Setting OMP_NUM_THREADS specified for pytorch..."
		export OMP_NUM_THREADS=$((cores_per_socket*sockets_))
		export NANO_TF_INTER_OP=1
	else
		export OMP_NUM_THREADS=${cores_per_socket}
		export NANO_TF_INTER_OP=$sockets_
	fi
```

Before the first `source bigdl-nano-init`, `ENABLE_TF_OPTS` is unset, so when source it for the first time, it will set OMP_NUM_THREADS to all cores, and set `NANO_TF_INTER_OP` to 1, after that it will set `ENABLE_TF_OPTS` to 1.

When source it for the second time, `ENABLE_TF_OPTS` is 1, so it will set `OMP_NUM_THREADS` to a single socket's cores, and set `NANO_TF_INTER_OP` to the number of sockets.

Anyway, it's a bug, I'll fix it.

**Comment by MeouSker77** [comment]
fixed it in #6670
